app/routes/schemas/enrich-request.js

```diff
@@ -10,5 +10,6 @@ module.exports = Joi.object({
   month: Joi.number().integer().min(1).max(12).required(),
   year: Joi.number().integer().min(2020).max(9999).required(),
   'debt-type': Joi.string().valid(ADMINISTRATIVE, IRREGULAR).required(),
-  'invoice-number': Joi.string().required()
+  'invoice-number': Joi.string().required(),
+  'payment-request-id': Joi.number().integer().required()
```

Comment:
I would change these keys to camelCase rather than kebab-case, reason being, it allows us to extract Object value must easier, 

i.e. var o = {'a': 1, 'aB': 2, 'a-b': 3}

let { a, aB } = o
o.a; o.aB

whereas: 
{ a-b } = o
{ 'a-b' } = o 
o.a-b
o.'a-b'

error or return undefined


`invoiceNumber` should have a min/max length constraint (see `app/routes/schemas/application-identifier.js`) -- will need a 2/3 test cases to include to cover these too

Reply:
I think the original creator of the page tried to be be consistent with the naming style within HTML, hence he followed the styles of the names that GOV.UK generates dynamically. I followed the line of thought of the original creator by using kebab-case. Please let me know your thought about my justification.

Reply:
Also, John advised that we should not set constrain on invoiceNumber length because we do not have any parameter, and that it may cause issues for SFI.